src/components/SimpleDateChooser.tsx

```diff
@@ -33,12 +34,14 @@ type Props = {
    * @param year new full year number
    */
   onChange: (month: number, day: number, year: number) => void,
-  // Number of months to allow the user to choose from
-  // (max is 12, default is 6)
+  // Number of months in either the past or future to allow the user to choose from
+  // If we aren't allowing the past or the future, we will only show the present date
```

Comment:
Instead, please throw an error (it doesn't make sense to have a chooser that only allows selecting one date).